spec/system/admin_customize_themes_spec.rb

```diff
@@ -83,70 +83,4 @@
       expect(ace_content.text).to eq("console.log('test')")
     end
   end
-
-  describe "when editing a theme setting of objects type" do
```

Comment:
Moved to https://github.com/discourse/discourse/pull/26133/files#diff-19603e878574022ae059d2815ea9dc5d3d38daa581d52e2de6563c1d50e8083a